locator/simple_strategy.cc

```diff
@@ -70,6 +70,9 @@ void simple_strategy::validate_options(const gms::feature_service&) const {
         throw exceptions::configuration_exception("SimpleStrategy requires a replication_factor strategy option.");
     }
     parse_replication_factor(it->second);
+    if (_uses_tablets) {
+        throw exceptions::configuration_exception("Tablet replication is not supported");
```

Comment:
"SimpleStrategy does not support tablet replication" perhaps? When I receive the configuration exception I don't have the full context, so it's best to capture it in the error message.